### ISSUE TITLE
Display table view for Fileclass not working

### ISSUE THREAD

**Issue by koenbunders** [mdelobelle/metadatamenu#173]
When I right click on my Books fileClass and then click 'Display Books table view' I get this:

Evaluation Error: TypeError: initialOptions.split is not a function
at new MultiSuggestModal (plugin:metadata-menu: 4422:47 )
at MultiField.createDvField (plugin:metadata-menu: 4682:20 )
at createDvField (plugin:metadata-menu: 11424:18)
at fieldModifier (plugin:metadata-menu: 11462:9 )
at eval (plugin:metadata-menu: 11568:41 )
at eval (eval at <anonymous> (plugin:dataview), <anonymous>: 6: 9)
at Array. map (anonymous>)
at Proxy.map (plugin:dataview: 9420:39 )
at eval (eval at <anonymous> (plugin:dataview), <anonymous> :4:6 )
at DataviewInlineApi.eval (plugin:dataview: 19673:16)

any idea what is causing it and how to solve it?

**Comment by mdelobelle** [comment]
good catch! it's a bug. I will correct that in the next version.
It occurs when you use "Multi Select" with some [[link]] in it. If you have a note with a such afield containing only one [[link]], the rendering of the table crashes.

Though I would suggest to use Multi File if you want to multi select files rather than the "multi select" type (you can filter them with a dataviewJs query)

**Comment by mdelobelle** [comment]
fixed in 0.3.7

**Comment by maksim77** [comment]
I have similar problem... 
I did not start a separate problem because the description fits exactly.

```
Evaluation Error: TypeError: p[this.field.name].split is not a function
    at MultiField.createDvField (plugin:metadata-menu:4754:44)
    at createDvField (plugin:metadata-menu:13044:18)
    at fieldModifier (plugin:metadata-menu:13082:9)
    at eval (plugin:metadata-menu:13191:41)
    at eval (eval at <anonymous> (plugin:dataview), <anonymous>:27:9)
    at Array.map (<anonymous>)
    at Proxy.map (plugin:dataview:9424:39)
    at eval (eval at <anonymous> (plugin:dataview), <anonymous>:22:6)
    at DataviewInlineApi.eval (plugin:dataview:19734:16)
    at evalInContext (plugin:dataview:19735:7)
```
